### PR TITLE
Fsa-123 story review

### PR DESCRIPTION
**Pull request by lunaoll**

### CODE REVIEW

**Comment by danielmihai8888** [comment]
Sounds good.

File: src/main/java/luna/week2/Question1.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package luna.week2;
+
+public class Question1 {
+
+    public static void main(String[] args) {
```
Comment (danielmihai8888) [inline src/main/java/luna/week2/Question1.java:5]: Maybe we can change the method name

**Review by danielmihai8888 — COMMENTED** [review]
Looks good.

File: src/main/java/luna/week2/Question1.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        System.out.println("test case 1 added here");
```
Comment (lunaoll) [inline src/main/java/luna/week2/Question1.java:6]: method name can be changed to compare :)

Comment (danielmihai8888) [reply src/main/java/luna/week2/Question1.java:6]: What is redundant?

File: src/main/java/luna/week2/Question1.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+
+        System.out.println("second test case implemnted");
+    }
```
Comment (lunaoll) [inline src/main/java/luna/week2/Question1.java:10]: this line seems redundant we can delete it

**Pull request closed without merging**